ios/Sources/MediaFeature/MediaSection.swift

```diff
@@ -12,6 +12,8 @@ public struct MediaSection: View {
 
     enum ViewAction {
         case showMore
+        case tap(FeedContent)
+        case favorite(String)
```

Comment:
Please fix with `isFavorited` as well 🙏🏼 
https://github.com/DroidKaigi/conference-app-2021/pull/528/files#diff-648db098a3fbc105524a38e3c277d8223ae1e87dd381df9a4646c5def825a893R7

Reply:
`isFavorite` is little weird for action name. is it okay with name `tapFavorite` ?

Reply:
Yes, action name should be `tapFavorite`. 
What I meant is closure argument.

Reply:
Fixed 🙏